src/python/sdk/examples/trade.py

```diff
@@ -0,0 +1,533 @@
+from os import environ
+from typing import Optional, Tuple
+
+from aptos_sdk.account import Account
+from aptos_sdk.account_address import AccountAddress
+from aptos_sdk.bcs import Serializer, encoder
+from aptos_sdk.client import FaucetClient, RestClient
+from aptos_sdk.transactions import EntryFunction, ModuleId
+from aptos_sdk.type_tag import StructTag, TypeTag
+from econia_sdk.entry.market import (
+    cancel_all_orders_user,
+    change_order_size_user,
+    place_limit_order_user_entry,
+    place_market_order_user_entry,
+    register_market_base_coin_from_coinstore,
+)
+from econia_sdk.entry.user import (
+    deposit_from_coinstore,
+    register_market_account,
+)
+from econia_sdk.lib import EconiaClient, EconiaViewer
+from econia_sdk.types import Restriction, SelfMatchBehavior, Side
+from econia_sdk.view.market import get_price_levels
+from econia_sdk.view.registry import (
+    get_market_id_base_coin,
+    get_market_registration_events,
+)
+from econia_sdk.view.resource_account import get_address
+from econia_sdk.view.user import (
+    get_cancel_order_events,
+    get_fill_events,
+    get_market_account,
+    get_place_limit_order_events,
+)
+
+"""
```

Comment:
> 3\. After step 3 add in another step saying that you need to switch the `econia` named address in Move.toml to `_`: this will get the tutorial follower familiar with using named addresses

Per recent offline team discussion disregard this, we will be using `_` (stand by for my PR to main that updates this and also some of the build scripting)

Reply:
done